internal/olm/operator/registry/index_image.go

```diff
@@ -451,7 +451,7 @@ func (c IndexImageCatalogCreator) UpdateCatalog(ctx context.Context, cs *v1alpha
 		// check if index image adopts File-Based Catalog or SQLite index image format
 		isFBCImage, err := fbcutil.IsFBC(ctx, c.IndexImage)
 		if err != nil {
-			return fmt.Errorf("error in upgrading the bundle %q that was installed traditionally", c.BundleImage)
+			return fmt.Errorf("unable to determine if index image adopts File-Based Catalog or SQLite format: %v", err)
```

Comment:
onsider including the index image in the error message. e.g.
`fmt.Errorf("unable to determine if index image %q adopts File-Based Catalog or SQLite format: %v", c.IndexImage, err)`

Reply:
if so, also update here https://github.com/operator-framework/operator-sdk/blob/d2366bef634034fa3179629cfef3b7d68cde1727/internal/olm/operator/registry/index_image.go#L429-L433